solutions/instances/DA-types.md

```diff
@@ -88,3 +88,38 @@ cloud_logs_policies = [
   }
 ]
 ```
+
+## Metric Router Routes <a name="metric_router_routes"></a>
+
+The `metric_router_routes` input variable allows you to provide a route that will be configured in the Metric Routing. Refer [here](https://cloud.ibm.com/docs/metrics-router?topic=metrics-router-about) for more information.
+
+- Variable name: `metric_router_routes`.
+- Type: An object
+- Default value: `null`.
+
+### Options for metric_router_routes
+
+  - `name` (required):  The name of the route.
+  - `rules` (required): The routing rules that will be evaluated in their order of the array.
+    - `action` (optional): The action if the inclusion_filters matches, default is send action.
+    - `inclusion_filters` (required): A list of conditions to be satisfied for routing metrics to pre-defined target.
+    - `targets` (required): The target uuid for a pre-defined metrics router target.
+
```

Comment:
- for rules, we should say the maximum number of rules allowed if there is any limit.
- we can add allowed values for action.
- inclusion_filters is an object, we should talk about its structure too.

We can link with provider link too as it has more details.